src/units.cpp

```diff
@@ -431,7 +431,7 @@ void updateUnitsMap(const UnitsPtr &units, UnitsMap &unitsMap, double exp = 1.0)
     if (units->isBaseUnit()) {
         auto found = unitsMap.find(units->name());
         if (found == unitsMap.end()) {
-            unitsMap.emplace(units->name(), 1.0);
+            unitsMap.emplace(units->name(), exp);
```

Comment:
If we're dealing with a base unit then shouldn't the exponent be 1.0 all the time?  I'm not sure which way around is better ... ho hum.

Reply:
I guess it should be `1.0`, but to have `exp` is not only the same in this particular case, but it also makes things look "correct" (when looking at the code). So, I would leave it as-is.